torch/csrc/Module.cpp

```diff
@@ -175,6 +176,35 @@ static PyObject * THPModule_crashIfATenASAN(PyObject *module, PyObject *arg) {
   return THPUtils_packInt32(at::_crash_if_asan(THPUtils_unpackInt(arg)));
 }
 
+static PyObject * THPModule_vitals_enabled(PyObject *module, PyObject *args) {
+  if (at::vitals::torchVitalEnabled()) {
+    Py_RETURN_TRUE;
+  }
+  Py_RETURN_FALSE;
+}
+
+static PyObject * THPModule_set_vital(PyObject *module, PyObject *args)
+{
+  Py_ssize_t num_args = args ? (Py_ssize_t) PyTuple_Size(args) : 0;
+  THPUtils_assert(num_args == 3, "expected exactly 3 arguments");
+
+  if (!at::vitals::torchVitalEnabled()) {
+    Py_RETURN_NONE;
+  }
+
+  PyObject *name = PyTuple_GET_ITEM(args, 0);
+  PyObject *attr = PyTuple_GET_ITEM(args, 1);
+  PyObject *value = PyTuple_GET_ITEM(args, 2);
+  std::string name_str = THPUtils_unpackString(name);
+  std::string attr_str = THPUtils_unpackString(attr);
+  std::string value_str = THPUtils_unpackString(value);
+  if (at::vitals::VitalsAPI.setVital(name_str, attr_str, value_str)) {
+    Py_RETURN_TRUE;
+  }
+  // Failed to match any supported vital sign, or vital sign was destructed already.
```

Comment:
this looks like stale comment, setVital will return false only if env variable is not set?

Reply:
Thanks for the catch! Yes.